### PR TITLE
Particle File Vault

### PR DESCRIPTION
**Pull request by folex**

### CODE REVIEW

File: crates/server-config/src/services_config.rs [hunk 1]
```diff
@@ -35,6 +35,9 @@ pub struct ServicesConfig {
     pub modules_dir: PathBuf,
     /// Dir to persist info about running services
     pub services_dir: PathBuf,
+    /// Dir to store directories shared between services
+    /// in the span of a single particle' execution
```
Comment (justprosh) [inline crates/server-config/src/services_config.rs:39]:
```suggestion
    /// in the span of a single particle execution  
```

File: crates/service-modules/Cargo.toml [hunk 2]
```diff
@@ -10,3 +10,7 @@ fs-utils = { path = "../fs-utils" }
 air-interpreter-wasm = "0.10.5"
 
 eyre = "0.6.5"
+serde = { version = "1.0.118", features = ["derive"] }
+serde_json = "1.0.64"
+blake3 = "0.3.8"
+faster-hex = "0.6.0"
```
Comment (justprosh) [inline crates/service-modules/Cargo.toml:16]:
```suggestion
faster-hex = "0.6.0"

```

File: crates/test-utils/src/utils.rs [hunk 3]
```diff
@@ -1,9 +1,8 @@
-pub use particle_modules::Hash;
-pub use particle_modules::{AddBlueprint, Dependency, ModuleRepository};
 
-use fluence_faas::TomlFaaSNamedModuleConfig;
 use host_closure::Args;
 use ivalue_utils::IValue;
+
+
```
Comment (justprosh) [inline crates/test-utils/src/utils.rs:5]:
```suggestion
```

File: particle-node/tests/builtin.rs [hunk 4]
```diff
@@ -46,6 +51,8 @@ struct NodeInfo {
 
 #[test]
 fn identify() {
+    enable_logs();
```
Comment (justprosh) [inline particle-node/tests/builtin.rs:54]:
```suggestion
```

File: particle-node/tests/file_share/Cargo.toml [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+[[bin]]
+name = "file_share"
+path = "src/main.rs"
+
+[workspace]
+
+[dependencies]
+fluence = "0.6.9"
+rand = "0.8.4"
+
```
Comment (justprosh) [inline particle-node/tests/file_share/Cargo.toml:16]:
```suggestion
```

File: crates/connected-client/src/lib.rs [hunk 6]
```diff
@@ -1,5 +1,6 @@
+#![feature(once_cell)]
```
[the remark this answers is not in the document]
Comment (folex) [reply crates/connected-client/src/lib.rs:1]: Yes, `std::lazy::Lazy` requires it

**Review by justprosh — APPROVED** [review]

**Pull request merged**